app/scripts/controllers/lineitemexportmodal.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('bulbsCmsApp')
+  .controller('LineitemexportmodalCtrl', function ($scope, $http, moment) {
+    var now = moment().tz('America/Chicago');
```

Comment:
Use the [`TIMEZONE_NAME` constant](https://github.com/theonion/bulbs-cms/blob/1dd1d175b8b5757dcc5169d8dbae42c4e4f54411/app/scripts/app.js#L119) as [in this use case](https://github.com/theonion/bulbs-cms/blob/1dd1d175b8b5757dcc5169d8dbae42c4e4f54411/app/scripts/filters/tzdate.js#L4).